app/views/admin/profiles/admin_profiles/index.html.erb

```diff
@@ -0,0 +1,39 @@
+<%= render TabsComponent.new do |c| %>
+  <%= c.tab(path: admin_admin_profiles_path) do %>
+    Admin Profiles
```

Comment:
I'm not sure we want to use the "profile" terminology on the frontend

Reply:
Done